sigma/backends/loki/loki.py

```diff
@@ -618,7 +618,7 @@ def update_parsed_conditions(
         ):
             if (
                 isinstance(condition.value, SigmaString)
-                and (self.case_insensitive or condition.value.contains_special())
+                and (not self.case_sensitive or condition.value.contains_special())

```

Comment:
Code comment for this function could do with updating to reflect the change.

Reply:
Addressed in 20fac6a I think that's the bit of the comment that you're referring to?